leie/README.mdwn

```diff
@@ -52,19 +60,17 @@ distro's packaged version:
     $ apt-get install sqlite3
 
 You'll want goose to migrate the db up and down, which requires
-golang.  If you don't have golang, you'll need to install it, set your
-GOPATH and add GOPATH to your local PATH.  Here is how you might do
-that on a Debian box:
+golang.  If you don't have golang, you'll need to install it and
+set your GOPATH.  Here is how you might do that on a Debian box:
 
     $ apt-get install golang
-    $ mkdir -p /usr/local/share/golang
-    $ export GOPATH=/usr/local/share/golang
-    $ export PATH=$PATH:$GOROOT/bin
+    $ mkdir -p {path-to-cavetl}/leie/golang
+    $ export GOPATH={path-to-cavetl}/leie/golang
 
-Or, if you don't want to install goose as a superuser:
+Or, if you want to install goose as a superuser:
 
-    $ mkdir -p {path-to-psm}/etl/golang
-    $ export GOPATH={path-to-psm}/etl/golang
+    $ mkdir -p /usr/local/share/golang
+    $ export GOPATH=/usr/local/share/golang
```

Comment:
I propose we drop the superuser section altogether! How does that sound to you?

Reply:
Sounds fine to me.  I'll update the PR with that change.